### PR TITLE
Fix zeppelin support for tables

### PR DESCRIPTION
**Pull request by DarthMax**

### CODE REVIEW

File: spark-cypher-testing/src/test/scala/org/opencypher/spark/impl/acceptance/MultipleGraphBehaviour.scala [hunk 1]
```diff
@@ -102,9 +103,10 @@ class MultipleGraphBehaviour extends CAPSTestSuite with ScanGraphInit {
         |RETURN GRAPH""".stripMargin
 
     val result = testGraph1.cypher(query)
-    result.records.toMaps shouldBe empty
+    result.getRecords should be(None)
```
Comment (s1ck) [inline spark-cypher-testing/src/test/scala/org/opencypher/spark/impl/acceptance/MultipleGraphBehaviour.scala]: much nicer API usage then before

File: okapi-relational/src/main/scala/org/opencypher/okapi/relational/api/planning/RelationalCypherResult.scala [hunk 2]
```diff
@@ -40,13 +40,16 @@ case class RelationalCypherResult[T <: Table[T]](
 
   override type Graph = RelationalCypherGraph[T]
 
-  override def getGraph: Option[Graph] = maybeRelational.map(_.graph)
-
-  override def getRecords: Option[RelationalCypherRecords[T]] = {
-    maybeRelational.map(op => session.records.from(op.header, op.table, op.returnItems.map(_.map(_.name))))
+  override def getGraph: Option[Graph] = maybeRelational.flatMap {
```
Comment (s1ck) [inline okapi-relational/src/main/scala/org/opencypher/okapi/relational/api/planning/RelationalCypherResult.scala]: didn't think about that before :see_no_evil: nice fix!

File: okapi-api/src/main/scala/org/opencypher/okapi/api/util/ZeppelinSupport.scala [hunk 3]
```diff
@@ -47,12 +46,14 @@ object ZeppelinSupport {
     /**
       * Visualizes the result in Zeppelin.
       * If the result contains a graph, it is shown as a network (see [[ZeppelinSupport.ZeppelinGraph#printGraph]]).
-      * If the result contains a tabular result, they are visualized as a table (see [[ZeppelinSupport.ZeppelinRecords#printTable]]).
+      * If the result contains a tabular result, they are:
+      *   - visualized as a graph if the result only contains nodes and relationships (see [[ZeppelinSupport.ZeppelinRecords#printGraph]])
+      *   - TODO: visualized as a table if the result contains non entity values (see [[ZeppelinSupport.ZeppelinRecords#printTable]])
```
Comment (s1ck) [inline okapi-api/src/main/scala/org/opencypher/okapi/api/util/ZeppelinSupport.scala]: Is there already an issue for that? Sounds rather useful

Comment (DarthMax) [reply okapi-api/src/main/scala/org/opencypher/okapi/api/util/ZeppelinSupport.scala]: This is rather hard to accomplish with the current CypherTable API. There is no way to get information about cypher types of the logical layout.

**Pull request merged**